code/03_create-figures.R

```diff
@@ -359,4 +359,54 @@ ggsave(here("markdown", "figures", "figure-a1-4.png"),
        units = "cm", device = "png", dpi = 600)
 
 
+### 12 - Figure A3.1 - Old Methodology Comparison ----
+
+figa31 <-
+  
+  basefile %>%
+  summarise_data(include_years = "all", format_numbers = FALSE) %>%
+  rename(qom_new = qom) %>%
+  
+  left_join(basefile %>%
+            mutate(los = los_old) %>%
+            summarise_data(include_years = "all", format_numbers = FALSE) %>%
+            rename(qom_old = qom) %>%
+            select(fy, qom_old)) %>%
+  
+  pivot_longer(cols = c("qom_old", "qom_new"),
+               names_to = "method",
+               values_to = "qom") %>%
+  
+  mutate(method = case_when(
+    method == "qom_old" ~ str_wrap("Old measure", width = 12),
+    method == "qom_new" ~ str_wrap(paste0("New measure (including ",
+                                      "care home activity)"),
+                                   width = 12)
+  )) %>%
+  
+  ggplot(aes(x = fy, y = qom, group = method)) +
+  geom_line(aes(linetype = method), colour = "#004785") +
+  theme(panel.background = element_blank(),
+        panel.grid.major.x = element_blank(),
+        panel.grid.major.y = element_blank(),
+        axis.title.x = element_text(size = 8, face = "bold"),
+        axis.title.y = element_text(size = 8, face = "bold"),
+        axis.text = element_text(size = 8),
+        axis.text.x = element_text(angle = 90),
+        legend.title = element_blank(),
+        strip.text = element_text(size = 8, hjust = 1),
+        strip.background = element_blank()) +
```

Comment:
I think we can remove these two lines as they are for facet plots as I know? I tried to remove them and the plot looks fine.